packages/logs/src/boot/logsPublicApi.ts

```diff
@@ -28,7 +29,135 @@ export interface LoggerConfiguration {
   context?: object
 }
 
-export type LogsPublicApi = ReturnType<typeof makeLogsPublicApi>
+export interface LogsPublicApi extends PublicApi {
```

Comment:
**💬 suggestion:** ‏Use this type as return type of `makeLogsPublicApi` so we ensure the returned value matches the typing

Reply:
Oops I forgot 🤦‍♂️